source/data-api/authenticate.txt

```diff
@@ -0,0 +1,152 @@
+.. _data-api-authenticate-requests:
+
+==============================
+Authenticate Data API Requests
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Data API endpoints generally require that incoming requests include
+authentication information for the user calling the endpoint. This lets
+the endpoint enforce rules and validate document schemas for each
+request.
+
+Users can authenticate through any of the following authentication
+providers:
+
+- :ref:`Email/Password <email-password-authentication>`
+
+- :ref:`API Key <api-key-authentication>`
+
+- :ref:`Custom JWT <custom-jwt-authentication>`
+
+.. note::
+   
+   You must :ref:`enable an authentication provider
+   <authentication-providers>` before users can authenticate with
+   it.
+
+This guide shows you how to include authentication information in Data
+API requests.
```

Comment:
somewhere round here think it'd be good to add a line saying that those credentials can either be passed via credential header or a bearer auth token to set up rest of page.